common.yml

```
@@ -984,67 +984,67 @@ translations:
     ###########################################################################
 
     - key: tools_others.libraries
-      t: Libraries
+      t: Bibliotecas
     - key: tools_others.libraries.description
-      t: Which libraries do you regularly use?
+      t: Quais bibliotecas você usa regularmente?
     - key: tools_others.libraries.others
-      t: Other Libraries
+      t: Outras bibliotecas
     - key: tools_others.libraries.others.description
-      t: Other answers (freeform field).
+      t: Outras respostas (campo de texto livre).
 
     - key: tools_others.text_editors
-      t: Text Editors
+      t: Editores de texto.
     - key: tools_others.text_editors.description
-      t: Which text editor(s) do you regularly use?
+      t: Quais editor (es) de texto você usa regularmente?
     - key: tools_others.text_editors.others
-      t: Other Text Editors
+      t: Outros editores de texto.
     - key: tools_others.text_editors.others.description
-      t: Other answers (freeform field).
+      t: Outras respostas (campo de texto livre).
 
     - key: tools_others.utilities
-      t: Utilities
+      t: Serviços de utilidade pública
     - key: tools_others.utilities.description
-      t: Which utilities or tools do you regularly use?
+      t: Quais serviços de utilidade pública ou ferramentas você usa regularmente?
     - key: tools_others.utilities.others
-      t: Other Utilities
+      t: Outros serviços de utilidade pública
     - key: tools_others.utilities.others.description
-      t: Other utilities (freeform field).
+      t: Outros serviços de utilidade pública (campo de texto livre).
 
     - key: tools_others.browsers
-      t: Browsers
+      t: Navegadores
     - key: tools_others.browsers.description
-      t: Which browser(s) do you primarily work in during initial development?
+      t: Qual (is) navegador (es) você usa principalmente durante o desenvolvimento inicial?
     - key: tools_others.browsers.others
-      t: Other Browsers
+      t: Outros navegadores
     - key: tools_others.browsers.others.description
-      t: Other answers (freeform field).
+      t: Outras respostas (campo de texto livre).
 
     - key: tools_others.build_tools
-      t: Build Tools
+      t: Ferramentas de build
     - key: tools_others.build_tools.description
-      t: Which build tools do you use?
+      t: Quais ferramentas de build você usa?
     - key: tools_others.build_tools.others
-      t: Other Build Tools
+      t: Outras ferramentas de build
     - key: tools_others.build_tools.others.description
-      t: Other answers (freeform field).
+      t: Outras respostas (campo de texto livre).
 
     - key: tools_others.non_js_languages
-      t: Non-JavaScript Languages
+      t: Linguagens de programação não baseadas em JavaScript
```

Acho que aqui é não é necessariamente baseadas em js. acho que é a tradução mais literal
```suggestion
      t: Linguagens de programação não JavaScript
```

ou 
```suggestion
      t: Linguagens de programação diferentes de JavaScript
```

---

Você tem razão, conferi no site aqui! Gostei da segunda sugestão, valeu